packages/core/src/lib/implementation/execute-plugin.ts

```diff
@@ -142,9 +142,8 @@ export async function executePlugins(
 
   progressBar?.endProgress('Done running plugins');
 
-  const errorsCallback = ({ reason }: PromiseRejectedResult) => {
-    console.error(reason);
-  };
+  const errorsCallback = ({ reason }: PromiseRejectedResult) =>
+    reason as string;
```

Comment:
How is it an "`errorsCallback`" if it now returns a string instead of logging something? :raised_eyebrow:

Reply:
good point

Reply:
done!